src/schema/v2/me/index.ts

```diff
@@ -142,7 +142,7 @@ export const LengthUnitPreference = new GraphQLEnumType({
 export const meType = new GraphQLObjectType<any, ResolverContext>({
   name: "Me",
   interfaces: [NodeInterface],
-  fields: {
+  fields: () => ({
```

Comment:
This was needed to avoid the circular dependency as specified here https://github.com/graphql/graphql-js/issues/373#issuecomment-215913280

Reply:
hm, not sure how it could cause this issue, but glad that you found solution

Reply:
it was a pain to figure out 😄